### DISCUSSION TITLE
How can I rewrite path from "/api/products" to "/mock/products.json" ?

### DISCUSSION THREAD

**Discussion by GF-Huang** [chimurai/http-proxy-middleware#547] [Q&A]
I'm using webpack for vuejs, this is my config, but not works:
```js
module.exports = {
  devServer: {
    proxy: {
      '/api': {
        target: 'http://localhost:8080',
        changeOrigin: true,
        pathRewrite(path, req) {
          return (path.replace('/api', '/mock') + '.json')
        }
      }
    }
  }
}
```

**Comment by adamduncan** [comment]
@GreatFireWall, had a similar use-case for a React app, and commented over on https://github.com/chimurai/http-proxy-middleware/issues/63#issuecomment-514332200. A similar config might do the trick for your Vue app?
